tools/tools/sysbuild/sysbuild.sh

```diff
@@ -623,14 +627,14 @@ cp /tmp/_sb_log ${SBMNT}/tmp
 b=`basename $0`
 if [ "x$c_arg" != "x" ] ; then
 	cp $c_arg ${SBMNT}/root
-	chroot ${SBMNT} sh /root/$0 -c /root/`basename $c_arg` $use_pkg chroot_script 
+	chroot ${SBMNT} sh /root/$0 -c /root/`basename $c_arg` $use_pkg chroot_script
 else
 	chroot ${SBMNT} sh /root/$0 $use_pkg chroot_script
 fi
 cp ${SBMNT}/tmp/_sb_log /tmp
 
 log_it create all mountpoints
-grep -v '^[ 	]*#' ${SBMNT}/etc/fstab | 
```

Comment:
while it's fine this time, in the future, can you separate out whitespace from non-whitespace changes?

Reply:
It is easy for me to force push a third commit with just whitespace changes. I'll do that. Thank you!